docs/docs/project-configuration-dev-test-build.mdx

```diff
@@ -148,3 +148,72 @@ yarn rw dev --debugPort 75028
 The flag passed in the CLI will always take precedence over your setting in the `redwood.toml`
 
 Just remember to also change the port you are attaching to in your `./vscode/launch.json`
+
+## Ignoring the `.yarn`-folder
+
+The Yarn folder contains the most recent Yarn executable that Redwood supports –
+which currently is the [recommended way](https://github.com/yarnpkg/yarn/issues/7741)
+to ensure things run smooth for everyone. From VSCode's perspective, this of course
+is just another folder containing code, so it will
+
+1. display it in the file browser
+2. include its contents in project-wide fulltext searches
+3. watch its contents for changes
+
+… which is something you *may* not need or want, depending on your personal preferences.
+
+Fortunately, all these aspects are configurable via `settings.json`. You have the
+choice of making these changes to your local redwood project's configuration
+found in `.vscode/settings.json` or globally (so they apply to other projects as
+well). For global changes, hit <kbd>F1</kbd> or <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
```

Comment:
It's Cmd+Shift+P on Mac. But I'm not sure how to visualize that in a nice way

Reply:
UTF-8 to the rescue.

### <kbd>⌘</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>

→ 70b2aeb42e796b82e00fa4cbf065e03dd4d4e84c